src/gluonts/model/rotbaum/_preprocess.py

```diff
@@ -429,9 +487,13 @@ def make_features(self, time_series: Dict, starting_index: int) -> List:
             list(
                 chain(
                     *[
-                        list(ent[0]) + list(ent[1].values())
+                        prefix + list(ent[0]) + list(ent[1].values())
                         for ent in [
-                            self._pre_transform(ts[starting_index:end_index])
+                            self._pre_transform(
+                                ts[starting_index:end_index],
+                                self.subtract_mean,
+                                self.count_nans,
+                            )
```

Comment:
@zoolhasson this change (and the next one below) appear to be fixing #2221 (please confirm). Do you think you could split this into a separate bugfix PR? This way we can backport the change to release branches. Thanks!